drivers/clock_control/clock_control_nrf2_fll16m.c

```diff
@@ -51,7 +44,6 @@ static const struct clock_options {
 		.mode = FLL16M_MODE_OPEN_LOOP,
 	},
 	{
-		.accuracy = FLL16M_CLOSED_LOOP_ACCURACY,
 		.mode = FLL16M_MODE_CLOSED_LOOP,
 	},
```

Comment:
The closed loop should have been removed. I think there was a task for this. You can remove it now and simplify the code.

Reply:
same